data/data/nutanix/pre-bootstrap/outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "image_id" {
+  value = nutanix_image.rhcos.id
+}
+
+output "bootstrap_ignition_image_id" {
+  value = nutanix_image.bootstrap_ignition.id
+}
+
+output "cluster_domain" {
+  value = var.cluster_domain
+}
+
+output "cluster_id" {
+  value = var.cluster_id
+}
```

Comment:
Why are these outputs needed? They are not dependent on anything created by the stage.

Reply:
Cleaned up some outputs (were useful for debug output)